### PR TITLE
How to contribute to our blog

### PR DESCRIPTION
**Pull request by sergiocruz**
Can somebody please proofread this before merging? 

- [x] @Atrophius 
- [x] @davidkpiano 

Thanks :)

### CODE REVIEW

**Comment by sergiocruz** [comment]
Please make comments on this PR under the lines you'd like to have changed.

But if you're lazy and would rather read the rendered version, [here's the file](https://github.com/OrlandoDevs/orlandodevs.github.io/blob/feature/contributing/CONTRIBUTING.md) :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+```
+> jekyll serve
+Configuration file: orlandodevs.github.io/_config.yml
+            Source: orlandodevs.github.io
+       Destination: orlandodevs.github.io/_site
+      Generating... done.
+ Auto-regeneration: enabled for 'orlandodevs.github.io'
+Configuration file: orlandodevs.github.io/_config.yml
+    Server address: http://0.0.0.0:4000/
+```
```
Comment (Atrophius) [inline CONTRIBUTING.md]: This whole section should provide the instructions for the Github Pages setup. They'll need Ruby installed. Then they can just `bundle install` and `bundle exec jekyll serve` to make sure the stuff they're doing always works with Github's Jekyll installation.

Comment (sergiocruz) [inline CONTRIBUTING.md]: Should I replace the `install jekyll` link with the github pages link instead? If so, what is that url again?

Comment (Atrophius) [inline CONTRIBUTING.md]: They really don't need to know anything other than "make sure Ruby is installed" and `bundle install` in the root of the repo.

Then instances of `jekyll serve` should be replaced with `bundle exec jekyll serve`

Comment (sergiocruz) [inline CONTRIBUTING.md]: Yeah it's just that "making sure ruby is installed, and running bundle exec" could be a huge hurdle for some beginners, especially on Windows. So I wanted to provide them with a link where they can find even more resources from there.

Basically shift that concern away from us so we can be more objective. Not sure how at this point though.

Comment (Atrophius) [inline CONTRIBUTING.md]: Here's the link to the instructions: https://help.github.com/articles/using-jekyll-with-pages/

Comment (sergiocruz) [inline CONTRIBUTING.md]: Perfecto!!!!

Comment (sergiocruz) [inline CONTRIBUTING.md]: @Atrophius I just pushed the new link.

**Comment by davidkpiano** [comment]
LGTM :ship:

**Comment by sergiocruz** [comment]
Just waiting for @Atrophius checkmark :white_check_mark:

**Comment by Atrophius** [comment]
:+1:

**Pull request merged**